shared/steps/assess/check_if_repo_content_exists.go

```diff
@@ -34,13 +35,14 @@ func CheckRepoFileContent(files ...File) features.Func {
 		}
 
 		for _, file := range files {
+			fmt.Println(fmt.Sprintf("shared.Owner %s file.Repository %s file.Path %s", shared.Owner, file.Repository, file.Path))
 			content, _, err := gclient.GetFile(shared.Owner, file.Repository, "main", file.Path)
 			if err != nil {
-				t.Fatal(fmt.Errorf("failed to find expected file: %w", err))
+				t.Fatal(fmt.Errorf("failed to find expected file %s/%s with error: %w", file.Repository, file.Path, err))
 			}
 
-			if file.Content != string(content) {
-				t.Fatalf("expected content did not equal actual: %s", string(content))
+			if strings.Compare(strings.TrimSpace(file.Content), strings.TrimSpace(string(content))) != 0 {
```

Comment:
The important part is that the content matches exactly. If we trim it won't be an exact match anymore.

Reply:
leading/trailing spaces/newlines does not need to be matched. It does not affect the content

Reply:
What do you mean leading/trailing space does not affect the content? It absolutely does. :D

Reply:
If it wouldn't you wouldn't have to remove them. ;)